### DISCUSSION TITLE
account scaduto, effetti su abilitazione commercialista e autorizzazioni app

### DISCUSSION THREAD

**Discussion by devtempgio** [fattureincloud/api#407] [Q&A] [answered]
Buongiorno,
cosa succede agli account scaduti che hanno autorizzato un account commercialista al proprio fatturato e hanno autorizzato un'app pubblica creata su Fatture in Cloud?
Devono riautorizzare sia il commercialista che l'applicazione che tramite OAuth aveva autorizzato?

Ci sono accounts scaduti che alla riattivazione non hanno più la nostra società come commercialista.

Grazie per l'aiuto

**Comment by valmoz** [comment, marked as answer]
Buongiorno @devtempgio ,
per quanto riguarda le app pubbliche, esse non vengono scollegate in caso di scadenza della licenza; può controllare comunque nella sezione Applicazioni Collegate del suo account Fatture in Cloud.

Per quanto riguarda i commercialisti le consiglio invece di contattare il nostro supporto clienti.

Grazie
Mauro
